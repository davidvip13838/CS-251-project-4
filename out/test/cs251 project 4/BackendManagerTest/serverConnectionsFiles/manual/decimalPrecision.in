3 3
0 1 14141.428535
0 2 14140.721481
1 2 2.236068
