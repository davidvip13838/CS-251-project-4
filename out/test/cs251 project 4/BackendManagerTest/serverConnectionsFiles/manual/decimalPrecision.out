14142.95755
